src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/AssetCategories/AssetCategory.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using EasyAbp.Abp.Trees;
+using EasyAbp.BookingService.AssetDefinitions;
+using EasyAbp.BookingService.Assets;
+using EasyAbp.BookingService.AssetSchedules;
+using EasyAbp.BookingService.PeriodSchemes;
+using JetBrains.Annotations;
+using Volo.Abp.Domain.Entities.Auditing;
+using Volo.Abp.MultiTenancy;
+
+namespace EasyAbp.BookingService.AssetCategories;
+
+public class AssetCategory : FullAuditedAggregateRoot<Guid>, ITree<AssetCategory>, IMultiTenant
+{
+    public virtual Guid? TenantId { get; protected set; }
+    
+    /// <summary>
+    /// Assets should have the same AssetDefinitionName as here. 
+    /// </summary>
+    [CanBeNull]
+    public virtual string AssetDefinitionName { get; protected set; }
+    
+    /// <summary>
+    /// Asset.PeriodSchemeId is preferred.
+    /// This value only affects assets in this category, but not assets in children.
+    /// Will fall back to <see cref="PeriodScheme"/> where the IsDefault property is <c>true</c> if the value here is <c>null</c>.
+    /// </summary>
+    public virtual Guid? PeriodSchemeId { get; protected set; }
+    
+    /// <summary>
+    /// This property determines whether assets can be occupied by default when there is no schedule created.
+    /// The SchedulePolicy of <see cref="Asset"/> is preferred.
+    /// This value only affects assets in this category, but not assets in children.
+    /// Will fall back to <see cref="AssetDefinition"/> if the value here is <c>null</c>.
+    /// </summary>
+    public virtual AssetSchedulePolicy? DefaultSchedulePolicy { get; protected set; }
+
```

Comment:
The `AssetCategory` should have `AdvanceBookingDays`  property too. When it is null, fall back to `AssetDefinition`.